crates/sui-network/src/randomness/mod.rs

```diff
@@ -639,25 +709,30 @@ impl RandomnessEventLoop {
                     shares.iter(),
                     &round.signature_message(),
                 );
+                let full_sig_cell_clone = full_sig_cell.clone();
 
                 // Record own partial sigs.
-                if !self.completed_sigs.contains(&(epoch, round)) {
+                if !self.completed_sigs.contains_key(&round) {
                     self.received_partial_sigs
                         .insert((round, self.network.peer_id()), partial_sigs.clone());
                     rounds_to_aggregate.push((epoch, round));
                 }
 
                 debug!("sending partial sigs for epoch {epoch}, round {round}");
-                spawn_monitored_task!(RandomnessEventLoop::send_partial_signatures_task(
-                    name,
-                    network,
-                    retry_interval,
-                    metrics,
-                    authority_info,
-                    epoch,
-                    round,
-                    partial_sigs
-                ))
+                (
+                    spawn_monitored_task!(RandomnessEventLoop::send_partial_signatures_task(
```

Comment:
send_round_signatures_task could be clearer now

Reply:
changed to `send_signatures_task`